app/test/big_blob_test.go

```diff
@@ -37,6 +38,8 @@ func (s *BigBlobSuite) SetupSuite() {
 	cfg := testnode.DefaultConfig().WithFundedAccounts(s.accounts...)
 	// purposefully bypass the configurable mempool check
 	cfg.TmConfig.Mempool.MaxTxBytes = appconsts.MaxTxSize * 2
+	// Use priority mempool for consistent error behavior
+	cfg.TmConfig.Mempool.Type = tmcfg.MempoolTypePriority
```

Comment:
yo @ninabarbakadze this test panics if we don't use the priority mempool (this pr change the default to CAT)

the failure panics due to the error being handled weird (you mentioned CAT handled things differently from the priority)

do we need to fix this before the v6 release assuming we want to use CAT?

Reply:
there's some conversion failure

```
        --- FAIL: TestBigBlobSuite/TestBlobExceedsMaxTxSize/1_MiB_over_the_max_blob_size (0.05s)
            iface.go:275: test panicked: interface conversion: error is *status.Error, not *user.BroadcastTxError
 ```
 
 the TxClient can't panic correct?

Reply:
No, it should not panic and yes, we should fix before v6 or at least look into it to make sure it won't break for the users at least. I'm already on it.